fe/src/components/Login.vue

```diff
@@ -34,12 +35,17 @@ export default {
   },
   methods: {
     submitForm() {
-      // not sure what to do here 
-      var res = httpService.post('api/user/create/', {email:'test@email.ca', password:'password'});
-      console.log(res)
-      console.log(res.data)
 
-    }  	
+      this.$refs['LoginForm'].validate((valid) => {
+        if(valid){
+          alert('Logging in')
+        }else{
+          alert('not logging in')
+        }
+      });
+      var res = httpService.post('api/user/create/', this.userCredentials)
```

Comment:
one thing i try to do is create a base http service that isn't used in a vue component, but by other services. so the `user.js` service does this bit (with the path as a constant in that service) and then in the code you call `userService.login(this.userCredentials)`

do that

Reply:
done